frontend/app/src/ui/mod.rs

```diff
@@ -28,7 +28,7 @@ pub struct UI {
     seed: u32,
     snapshot: Option<Snapshot>,
     uninterpolated_snapshot: Option<Snapshot>,
-    pending_snapshots: VecDeque<Snapshot>,
+    snapshots: Vec<Snapshot>,
```

Comment:
We no longer need this to be a Deque. I was able to shave off 19% of used memory in a benchmark with this change.